code/__HELPERS/mobs.dm

```diff
@@ -319,9 +319,27 @@ Proc for attack log creation, because really why not
 	if(progbar)
 		qdel(progbar)
 
-/proc/isabductor(A)
+/proc/is_species(A, species_datum)
 	. = FALSE
 	if(ishuman(A))
 		var/mob/living/carbon/human/H = A
-		if(H.dna && istype(H.dna.species, /datum/species/abductor))
+		if(H.dna && istype(H.dna.species, species_datum))
 			. = TRUE
+
+/proc/isabductor(A)

```

Comment:
goof these procs are not called on hot paths, I wouldn't worry too much about it

they could be macro defines however which would eliminate the extra proc call